scripts/test-integration.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+cp ./config/config.dev.json.example ./config/config.dev.json
```

Comment:
Couple comments on this script:

**Relative vs. Absolute paths**
You're using relative paths, it's safer to use an absolute path that doesn't rely on the current working directly. As an example, the `remote-dev.sh` script [get's the directory that the script is in and sets `$_dir`](https://github.com/fusor/mig-ui/blob/master/scripts/remote-dev.sh#L2). Then you can anchor off of this location to refer to other files by their absolute locations:

```
#!/bin/bash
_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
projectRoot="$_dir/.." # Refers to the root of the git project
cp "$projectRoot/config/config.dev.json.example" "$projectRoot/config/config.dev.json"
# etc..
```
**Overwriting developer config**
` cp ./config/config.dev.json.example ./config/config.dev.json`

This is dangerous because you're going to overwrite any changes a developer has made to their config file if they choose to run this test on their local machine.

You should check to make sure that the file is not present first before copying, something like:
```
if [[ ! -f "$FILE" ]]; then
    echo "$FILE does not exist"
fi
```

So the overall I'd recommend this `test-integration.sh` to address the two issues: https://github.com/eriknelson/mig-ui/blob/andyvesel-add-tests/scripts/test-integration.sh

Reply:
Thanks for thorough reference, updated.